Doc/library/dataclasses.rst

```diff
@@ -430,8 +430,8 @@ Module contents
 
    Creates a new object of the same type as ``obj``, replacing
    fields with values from ``changes``.  If ``obj`` is not a Data
-   Class, raises :exc:`TypeError`.  If values in ``changes`` do not
-   specify fields, raises :exc:`TypeError`.
+   Class, raises :exc:`TypeError`.  If keys in ``changes`` are not correct
+   field names for the given dataclass, raises :exc:`TypeError`.
```

Comment:
I think I'd delete the word "correct", but it seems good enough either way.

Reply:
FWIW, I think it would be better with that suggestion applied, @ericvsmith.

Reply:
kk